components/Header.js

```diff
@@ -52,15 +61,20 @@ const StyledHeader = styled.header`
   align-items: center;
 `;
 
-const StyledButton = styled.button`
+const StyledButtonOpen = styled.button`
   background-color: transparent;
   border: none;
 `;
+const StyledButtonSignOut = styled.button`
+  background-color: transparent;
+  border: none;
+  margin-top: 3px;
+`;
```

Comment:
please use only one button component (e.g. `StyledButton` as before) and control its appearance by passing props. Normally it should not be necessary to have `margin-top` set for only one specific button. Maybe you find a better layout method (e.g. flexbox or grid) to prevent such individual pixel-stylings.

Reply:
done!